src/clj/puppetlabs/services/legacy_routes/legacy_routes_core.clj

```diff
@@ -46,21 +85,18 @@
   [handler mount-point api-version request]
   (let [{{environment :environment} :params
          uri :uri
-         query-string               :query-string} request
-        path-info (str "/" (-> (string/split uri #"/" 3) (nth 2)))
-        query-params (if (str/blank? query-string)
-                       {}
-                       (ring-codec/form-decode query-string))]
+         query-string :query-string} request
+        path-info (str "/" (-> (string/split uri #"/" 3) (nth 2)))]
```

Comment:
Bonus points for an idiomatic Clojure solution, maybe using [path-info](https://github.com/ring-clojure/ring/blob/1.3.1/ring-core/src/ring/util/request.clj#L66) from ring.util.request, or something similar?